feat/feature-flags.json

```diff
@@ -10,7 +10,7 @@
     },
     "new_header": {
       "status": {
-        "staging": "switchable",
+        "staging": "enabled",
```

Comment:
We could also probably keep the flag switchable, but set its defaultState to "on" (line 17). This way, we could still go back to the old design in staging using preferences

Reply:
Oh, and it's also the only way to get the "old_header" Playwright tests to pass.

Reply:
I pushed the change you suggested, Olga, so that the PR will be ready for the bug bash later (hopefully).

Reply:
Thank you, @sarayourfriend! I was also working on the adjustments necessary for the tests to pass, and I'll push them in a couple of minutes. It's mostly explicitly disabling the "new_header" flag for the "old" pages.